tests/src/test/java/alluxio/server/ft/journal/JournalShutdownIntegrationTest.java

```diff
@@ -178,15 +178,24 @@ public void singleMasterMountUnmountJournal() throws Exception {
 
   @Test
   public void multiMasterMountUnmountJournal() throws Exception {
-    MultiMasterLocalAlluxioCluster cluster = setupMultiMasterCluster();
-    UnderFileSystemFactory factory = mountUnmount(cluster.getClient());
-    // Kill the leader one by one.
-    for (int kills = 0; kills < TEST_NUM_MASTERS; kills++) {
-      cluster.waitForNewMaster(120 * Constants.SECOND_MS);
-      assertTrue(cluster.stopLeader());
+    MultiMasterLocalAlluxioCluster cluster = null;
+    UnderFileSystemFactory factory = null;
+    try {
+      cluster = new MultiMasterLocalAlluxioCluster(TEST_NUM_MASTERS);
```

Comment:
Should construction of a new cluster always be in the try block? We include it here but leave it outside the try for `MultiProcessClusters`.

Reply:
Most tests are using `before` and `after` methods for setting up and tearing clusters, which makes using try statements a bigger change.